code/modules/mob/living/basic/space_fauna/spider/spiderlings/spiderling_subtypes.dm

```diff
@@ -3,23 +3,81 @@
 /// Will differentiate into the base giant spider (known colloquially as the "guard" spider).
 /mob/living/basic/spiderling/guard
 	grow_as = /mob/living/basic/giant_spider
+	name = "guard spiderling"
+	desc = "Furry and brown, it makes you shudder to look at it. This one has sparkling red eyes."
+	icon_state = "guard_spiderling"
+	icon_dead = "guard_spiderling_dead"
+	menu_description = "Furry and brown, specializing in defense of the hive and other spides."
```

Comment:
This is a bad pattern, we should get this data from the grown spider and don't need it on the spiderling. Having to write the same information in two places is not good.

Reply:
That's what Sans did I'm just building on it by adding more spiderlings

Reply:
San did not do this because they did not have a menu description until you added it.